src/components/column-sorting/column-sorting.component.ts

```diff
@@ -7,7 +7,10 @@ import { ColumnSortingDirective, ColumnSortingIndicatorContext, ColumnSortingOrd
     selector: 'ux-column-sorting',
     templateUrl: './column-sorting.component.html',
     exportAs: 'ux-column-sorting',
-    changeDetection: ChangeDetectionStrategy.OnPush
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    host: {
+        '[class.ux-column-sorting-nosort]': `state === ColumnSortingState.NoSort`
+    }
```

Comment:
This doesn't work as intended, so at this point in the sprint I would recommend removing it and leaving the font weight at normal in all states.